src/test.js

```diff
@@ -4,350 +4,329 @@ const autodux = require('./');
 const id = autodux.id;
 const assign = autodux.assign;
 
-const createDux = () =>
+const createCounterDux = () =>
   autodux({
+    // The slice of state your reducer controls.
     slice: 'counter',
+
+    // The initial value for the slice of state.
     initial: 0,
+
+    // No need to implement switching logic, it's done for you.
     actions: {
-      increment: {
-        reducer: state => state + 1
-      },
+      // Shorthand definition of action and corresponding reducer.
+      increment: state => state + 1,
+
+      // Alternative definition of action and corresponding reducer.
       decrement: {
         reducer: state => state - 1
       },
+
+      // Another way to define action and reducer.
       multiply: {
-        create: id,
+        // Define custom mapping of action creator parameter to action payload.
+        create: ({ by }) => by,
+
         reducer: (state, payload) => state * payload
+      },
+
+      // Define action and reducer without custom mapping of action creator parameter to action payload.
+      randomize: {
+        reducer: (_, { max, min }) =>
+          Math.floor(Math.random() * (Math.floor(max) - Math.ceil(min) + 1)) +
+          Math.ceil(min)
       }
     },
+
     selectors: {
+      // No need to select the state slice, it's done for you.
       getValue: id,
-      getStore: (_, store) => store
+
+      // Get access to the root state in case you need it.
+      getSomething: (_, rootState) => rootState
     }
   });
 
-describe('autodux().slice', async assert => {
+const createWeirdDux = () => autodux();
```

Comment:
This should probably just throw, unless I'm missing something. Calling with no args wouldn't produce a very useful reducer. There's a `Throw` utility in RITEway that makes it very easy to test.

Reply:
Yes, calling `autodux` without argument seems to make no sense. Throwing an error here would result in a change within _src/index.js_, which I planned to introduce in another pull request. We can do it here though if you think it's OK.

Reply:
Yep, it's OK.

Reply:
This case is now [handled](https://github.com/ericelliott/autodux/pull/35/files#diff-f6c2ded91c9881cdc31b5bfcabb54c3bR354).